cloudsync_gdrive.py

```diff
@@ -337,6 +337,9 @@ def events(self) -> Generator[Event, None, None]:  # pylint: disable=too-many-lo
 
                 ohash = None
                 path = self._path_oid(oid, use_cache=False)
```

Comment:
wow this is super expensive... we double-tap gdrive on every event.  need to remove that stuff.   should just not include the path.